scripts/start-setupModpack

```diff
@@ -2,8 +2,11 @@
 
 set -e -o pipefail
 
-: "${REMOVE_OLD_MODS:=false}"
 : "${MODS_FILE:=}"
+: "${REMOVE_OLD_FILES:=false}"
+: "${REMOVE_OLD_FILES_DEPTH:=2} "
+: "${REMOVE_OLD_FILES_INCLUDE:={plugins,mods\}/*.jar,{plugins,mods\}/*-version.json}"
```

Comment:
These are name matching patterns not path matching. This isn't going to work as is. Instead `plugins` and `mods` need to be passed as the entry points.

Reply:
https://github.com/itzg/docker-minecraft-server/blob/d60e67e6f99318360fa765c23020e521a8221d34/scripts/start-utils#L215

https://github.com/itzg/mc-image-helper/blob/master/src/main/java/me/itzg/helpers/find/PathMatcherConverter.java#L11-L15

> [The following rules are used to interpret glob patterns:](https://docs.oracle.com/javase/7/docs/api/java/nio/file/FileSystem.html#getPathMatcher%28java.lang.String%29)
>
> The { } characters are a group of subpatterns, where the group matches if any subpattern in the group matches. The "," character is used to separate the subpatterns. Groups cannot be nested.

Reply:
Oh, understood. I think it'd be useful here if we could glob paths in `--name`.

I can see where in `mc-image-helper` the tree is being walked through instead of `getPathMatcher`.

Reply:
Sorry I should clarified in the first place. Yes, just needs an enhancement to mc-image-helper